StockWatch.Backend/src/main/java/com/henning/oden/java/StockAlert/Backend/entities/AuthenticationRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.henning.oden.java.StockAlert.Backend.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+
+@AllArgsConstructor
```

Comment:
Instead of all these annotations, can you use @Value to make it immutable. Strive for immutability in general

Reply:
Good idea! Thanks for pointing this option out.